.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: publish
+on: [workflow_dispatch]
```

Comment:
It should be automatic based on tag/release creation instead of manual dispatch

Reply:
Changed in [4c864e7](https://github.com/RyanModDev/EmuNO/pull/4/commits/4c864e78794f94d1e81bbbe092959cfa0a700779)